features/netsocket/nsapi_dns.cpp

```diff
@@ -27,10 +31,70 @@
 
 // DNS options
 #define DNS_BUFFER_SIZE 512
-#define DNS_TIMEOUT 5000
 #define DNS_SERVERS_SIZE 5
+#define DNS_RESPONSE_MIN_SIZE 12
+#define DNS_MAX_TTL 604800
+#define DNS_STACK_SERVERS_NUM 5
+#define DNS_QUERY_QUEUE_SIZE 5
+#define DNS_HOST_NAME_MAX_LEN 255
+
+#ifndef MBED_CONF_NSAPI_DNS_RESPONSE_WAIT_TIME
+#define MBED_CONF_NSAPI_DNS_RESPONSE_WAIT_TIME   5000
```

Comment:
Do we need these defaults? I just helped someone out who was confusing himself by trying to change lines like these, rather than the JSON file, to no effect.

Reply:
Removed those.